src/cpg_seqr_loader/alphagenome_config_template.toml

```diff
@@ -1,6 +1,104 @@
 # Template for the alphagenome config file
 [alphagenome_params]
-genes = ["ENSG00000123456", "ENSG00000234567", "ENSG00000345678"]
+genes = ["ENSG00000159640",
```

Comment:
This real content should be removed from the template - we have a separate repository for populated configuration files 
https://github.com/populationgenomics/production-pipelines-configuration